MediaBrowser.Plugins.Vimeo/VimeoAPI/Common/OAuthBase.cs

```diff
@@ -1,4 +1,4 @@
-﻿#if WINDOWS
+#if WINDOWS
```

Comment:
Why?

Reply:
I don't know.. I wanted to remove it but was afraid of breaking windows support in some way

Reply:
Meh. Removing.